adminSiteClient/CountryStandardizerPage.tsx

```diff
@@ -355,24 +355,18 @@ export class CountryStandardizerPage extends React.Component {
         const reader = new FileReader()
         reader.onload = (e) => {
             const csv = e?.target?.result
-            if (csv && typeof csv === "string")
-                parse(
-                    csv,
-                    {
-                        relax_column_count: true,
-                        skip_empty_lines: true,
-                        rtrim: true,
```

Comment:
there's a very slight behavior change between the [`csv-parse` options](https://csv.js.org/parse/options/) and [`papaparse` options](https://www.papaparse.com/docs#config) here:
* `skip_empty_lines` we have covered with `skipEmptyLines` (duh)
* `relax_column_count` is the default behavior of papaparse, meaning that CSV lines with less or more columns than the first line are allowed
* there is no equivalent for `rtrim`, which would trim right whitespace in `a ,b`, but not in `"a ",b`
  * I highly doubt that we rely on this in any way, especially since we don't also use `ltrim`?